workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Editor/Prefabs/PrefabPreprocessor.cs

```diff
@@ -75,20 +75,22 @@ private static void PreprocessPrefabs()
         private static bool IsBehaviourEnabledInEditor(Object obj)
         {
             return !ReferenceEquals(obj, null)
-                && EditorUtility.GetObjectEnabled(obj) == 1;
+                   && EditorUtility.GetObjectEnabled(obj) == 1;
         }
 
-        private static bool DoesBehaviourRequireReadersOrWriters(Type targetType)
+        private static bool DoesBehaviourNeedConditionalEnabling(Type targetType)
         {
-            return targetType
-                .GetFields(BindingFlags.Instance | BindingFlags.NonPublic | BindingFlags.Public)
-                .Any(field => Attribute.IsDefined(field, typeof(RequireAttribute), false));
+            return
```

Comment:
This is a potential target for caching the results of the check, since the `bool` result will never change.

Reply:
Good idea, will do